spec/models/qualifications_spec.rb

```diff
@@ -1,9 +1,107 @@
 require 'rails_helper'
 
 RSpec.describe Qualifications, type: :model do
-  describe 'QTS only' do
-    subject { Qualifications.new(profpost_flag: "recommendation_for_qts", is_pgde: false, is_fe: false) }
+  describe 'QTS' do
+    context "is recommendation_for_qts and is not pgde and is not further education" do
+      subject { Qualifications.new(profpost_flag: "recommendation_for_qts", is_pgde: false, is_fe: false) }
 
-    its(:to_a) { should eq([:qts])}
+      its(:to_a) { should eq([:qts])}
+    end
+  end
+
+  describe 'PGCE with QTS' do
+    context "is professional and is not pgde and is not further education" do
+      subject { Qualifications.new(profpost_flag: "professional", is_pgde: false, is_fe: false) }
+
+      its(:to_a) { should match_array([:qts, :pgce])}
+    end
+
+    context "is postgraduate and is not pgde and is not further education" do
+      subject { Qualifications.new(profpost_flag: "professional", is_pgde: false, is_fe: false) }
```

Comment:
@defong there are copy-paste errors in this spec meaning it's testing the same thing as the previous one. fixed in ad13cc3

Reply:
Cheers I was testing it to death